mod/wikirate/spec/set/right/cited_claims_spec.rb

```diff
@@ -14,7 +14,7 @@
       #create claim related to analysis but not cited
 
       claim_card = create_claim "whateverclaim",{"+company"=>{:content=>"[[#{@sample_company.name}]]"},"+topic"=>{:content=>"[[#{@sample_topic.name}]]"}}
-      sample_article = @sample_analysis.fetch :trait=>:wikirate_article, :new=>{}
+      sample_article = @sample_analysis.fetch :trait=>:overview, :new=>{}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.